src/conduit/tests/test_util.py

```diff
@@ -0,0 +1,24 @@
+"""Test app utilities."""
+
+from conduit.util import expandvars_dict
+
+
+def test_expandvars_dict() -> None:
+    """Test if config vars are properly parsed."""
+    settings = {
+        "really": "true",
+        "count": "1",
+        "url": "http://${HOST:-localhost}:${PORT:-8080}",
```

Comment:
Why is there a `-` sign after the colon?

Reply:
This is the way to define default values in shell.
```bash
echo ${FOO:-bar}
bar
```
```bash
FOO=foo
echo ${FOO:-bar}
foo
```
[Here is the GNU official doc](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) on shell parameter expansion. `expandvars` [works with all the string related expansion options](https://github.com/sayanarijit/expandvars/blob/ed58c0df5461b5b45d98ac377c3ffde0e334d468/tests/test_expandvars.py#L8-L99).

Reply:
nice :+1: